### PR TITLE
Fix release event and test event parsing with database

### LINKED ISSUE #536: Release event for build wrongly parsed
Sentry Issue: [RED-HAT-0P-2CV](https://sentry.io/organizations/red-hat-0p/issues/1595926482/?referrer=github_integration)

```
AssertionError: None
(10 additional frame(s) were not displayed)
...
  File "packit_service/worker/reporting.py", line 70, in set_status
    self.commit_sha, state, url, description, check_name, trim=True
  File "ogr/read_only.py", line 70, in readonly_func
    return func(self, *args, **kwargs)
  File "ogr/services/github/project.py", line 355, in set_commit_status
    trim=trim,
  File "ogr/services/github/flag.py", line 81, in set
    github_commit = project.github_repo.get_commit(commit)
  File "github/Repository.py", line 1453, in get_commit
    assert isinstance(sha, str), sha
```

Relevant release:
https://github.com/packit-service/hello-world/releases/tag/0.74

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Fix release event parsing. Fixes #536 
- Add tests for parsing events to the requre(=database) testsuite.

### CODE REVIEW

File: packit_service/models.py [hunk 1]
```diff
@@ -278,14 +282,14 @@ def get_or_create(
             if not project_release:
                 project_release = ProjectReleaseModel()
                 project_release.tag_name = tag_name
-                project_release.project_id = project.id
+                project_release.project = project
```
Comment (TomasTomecek) [inline packit_service/models.py:285]: interesting, so SA will populate project_id on its own?

Comment (lachmanfrantisek) [reply packit_service/models.py:285]: Yes, it should.

**Comment by lachmanfrantisek** [comment]
recheck

**Comment by lachmanfrantisek** [comment]
recheck

**Pull request merged**